CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing Guidelines
 * You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
-* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](http://www.systers.io/code-of-conduct.html).
+* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](http://www.systers.io/code-of-conduct/).
```

Comment:
The link is not working.
Correct link: https://github.com/systers/powerup-android/blob/develop/code_of_conduct.md